src/apps/api/serializers/leaderboards.py

```diff
@@ -105,3 +107,52 @@ def get_submissions(self, instance):
 
         submissions = submissions.order_by(*ordering, 'created_when')
         return SubmissionLeaderBoardSerializer(submissions, many=True).data
+
+
+class LeaderboardPhaseSerializer(serializers.ModelSerializer):
+    submissions = serializers.SerializerMethodField(read_only=True)
+    tasks = TaskSerializer(many=True, read_only=True)
+    columns = serializers.SerializerMethodField()
+
+    def get_columns(self, instance):
+        columns = Column.objects.filter(leaderboard=instance.leaderboard)
+        if len(columns) == 0:
+            raise serializers.ValidationError("No columns exist on the leaderboard")
+        elif len(columns) == 1:
+            return ColumnSerializer(columns).data
```

Comment:
you could simplify this whole conditional like this maybe:

```python
        if len(columns) == 0:
            raise serializers.ValidationError("No columns exist on the leaderboard")
        elif len(columns) >= 1:
            return ColumnSerializer(columns, many=len(columns) > 1).data
```

Reply:
I never thought of that. Neat!